src/Microsoft.Health.Dicom.Core/Features/ExtendedQueryTag/ExtendedQueryTagError.cs

```diff
@@ -23,7 +23,7 @@ public class ExtendedQueryTagError
             StudyInstanceUid = EnsureArg.IsNotNullOrWhiteSpace(studyInstanceUid);
             SeriesInstanceUid = seriesInstanceUid;
```

Comment:
By the way, should these errors now require study/series/instance uid?

Reply:
I think so, the instance table in our db has these fields as not null.